corehq/apps/app_manager/suite_xml/case_tile_templates/clinic_and_unit.xml

```diff
@@ -0,0 +1,96 @@
+<detail id="{detail_id}">
+  <title>
+    <text>
+        <locale id="{title_text_id}"/>
+    </text>
+  </title>
+
+  <field>
+    <style horz-align="left" vert-align="center" font-size="medium">
+        <grid grid-height="1" grid-width="12" grid-x="0" grid-y="0"/>
+    </style>
+    <header>
+      <text>
+        <locale id="{parent_case_header[locale_id]}"/>
+      </text>
+    </header>
+    <template form="{parent_case_header[format]}">
+      <text>
+        <xpath function="{parent_case_header[xpath_function]}">
+          {parent_case_header[variables]}
+        </xpath>
+      </text>
+    </template>
+  </field>
+
+  <field>
+    <style horz-align="left" vert-align="center" font-size="medium">
```

Comment:
This block shouldn't matter because the `width=0` on the `<template>` means it won't be displayed. But it isn't harming anything, either, it just means this section of the tile will be blank.